### DISCUSSION TITLE
Docs website

### DISCUSSION THREAD

**Discussion by vasucp1207** [zloirock/core-js#1182] [General]
Hey, @zloirock see your post about this project and I am really inspired by the amount of hard work you put into maintaining this project, as a beginner developer I want to contribute to open-source projects and I can start with building the docs site for the project and slowly getting familiar with the main source code also.

**Comment by zloirock** [comment]
It could be great, thanks. I'm thinking about creating a static website https://github.com/zloirock/core-js/issues/1053, after choosing an engine you could help.

**Comment by Rudxain** [comment]
I'm in! We can use #1095 as a "base". But I have to sync it with upstream. I'll update it after the static website has been chosen. Right now, I don't have time to eagerly (polling) keep it in sync, so I'm taking an event-driven (`async`) approach, haha
